appveyor.yml

```diff
@@ -1,5 +1,7 @@
 environment:
   matrix:
+  - TOXENV: "py37-xdist"
+  - TOXENV: "py27-xdist"
   - TOXENV: "py27"
   - TOXENV: "py37"
```

Comment:
I would argue that it's ok to remove py27 and py37 here then, but left it out for now.
It might be good to run everything with xdist on AppVeyor (but would need new tox factors).